megamek/src/megamek/client/ui/swing/dialog/AbstractUnitSelectorDialog.java

```diff
@@ -187,7 +187,10 @@ private void setUserPreferences() {
         ((DefaultRowSorter<?, ?>) tableUnits.getRowSorter()).sort();
 
         tableUnits.invalidate(); // force re-layout of window
-        pack();
+//        pack();
```

Comment:
Commented out pack?

Reply:
Same as the RandomArmyDialog, this dialog now remembers size&position and the pack had to go to make it work.